test-suite/asianoptions.cpp

```diff
@@ -651,7 +651,7 @@ void AsianOptionTest::testMCDiscreteGeometricAveragePriceHeston() {
 
     ext::shared_ptr<PricingEngine> engine =
         MakeMCDiscreteGeometricAPHestonEngine<LowDiscrepancy>(hestonProcess)
-        .withSamples(32767)
+        .withSamples(10000)
```

Comment:
Low-discrepancy generators require, by construction, a number of samples equal to 2^N-1 for some N.  You can use 8191 or 16383.

Reply:
> Low-discrepancy generators require, by construction, a number of samples equal to 2^N-1 for some N. You can use 8191 or 16383.

It makes sense, sorry I didn't notice it's using the low discrepency.